packages/common/src/audius-query/utils.ts

```diff
@@ -15,8 +16,18 @@ export const getKeyFromFetchArgs = (fetchArgs: any) => {
 
 export const selectCommonEntityMap = (
   state: CommonState,
-  kind?: Kind
+  kind?: Kind,
+  shallow?: boolean
 ): EntityMap | null => {
+  if (kind && shallow) {
```

Comment:
sick so basically we dont need to grab the whole state right?

Reply:
right and we only need to do this if the top level data is itself an entity. Without this step, we'd just get `useGetTrackById({ id: 123 }, { shallow: true }) returns { data: 123 }`